tools/mo/openvino/tools/mo/back/compress_quantized_weights.py

```diff
@@ -143,7 +143,7 @@ def quantize_data(fake_quantize: Node, dst_type: type, quantized_type: type, mod
         assert mode in ["signed", "unsigned"]
         i_min_value = -(levels // 2) if mode == "signed" else 0
 
-        i_min = mo_array([i_min_value], dtype=dst_type)
+        i_min = mo_array(i_min_value, dtype=dst_type) if not quantize.in_node(0).shape.size else mo_array([i_min_value], dtype=dst_type)
```

Comment:
can we use a scalar always here: `i_min = mo_array(i_min_value, dtype=dst_type)`? Then it will be always broadcastable to the shape of data from port 0.

Reply:
As far as I know, it was made especially for 1d transformation here. @mvafin please clarify whether it is still relevant.

Reply:
cc'ed @jane-intel